### PR TITLE
Minor documentation update

### PR DESCRIPTION
**Pull request by MaxKsyunz**
Callout distinction between original and current SQL engines.

Signed-off-by: Max Ksyunz <maxk@bitquilltech.com>

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by joshuali925 — APPROVED** [review]

File: docs/user/beyond/fulltext.rst [hunk 1]
```diff
@@ -14,6 +14,9 @@ Introduction
 
 Full-text search is for searching a single stored document which is distinguished from regular search based on original texts in database. It tries to match search criteria by examining all of the words in each document. In OpenSearch, full-text queries provided enables you to search text fields analyzed during indexing.
 
+This document describes the original SQL engine. It only applies to queries that cannot be handled by the current SQL engine. Documentation for the current engine can be found  `here <../dql/functions.rst#relevance>`_.
```
Comment (dai-chen) [inline docs/user/beyond/fulltext.rst:17]: Officially we use SQL engine v2 for current/new engine. Could you stick with it?

https://github.com/opensearch-project/sql/blob/main/docs/dev/NewSQLEngine.md

**Review by dai-chen — APPROVED** [review]
Minor comment. Thanks for the changes!

File: docs/user/beyond/fulltext.rst (hunk 1 above)
Comment (acarbonetto) [reply docs/user/beyond/fulltext.rst:17]: @MaxKsyunz What about: 
```
This document describes the original SQL engine. It only applies to queries that cannot be handled by `SQL engine V2 <../../dev/NewSQLEngine.md>`. Documentation for full-text search using SQL Engine V2 can be found  `here <../dql/functions.rst#relevance>`_.
```

Comment (dai-chen) [reply docs/user/beyond/fulltext.rst:17]: Looks good. Thanks!

**Review by joshuali925 — APPROVED** [review]

**Review by dai-chen — APPROVED** [review]

**Comment by MaxKsyunz** [comment]
Please see https://github.com/opensearch-project/sql/pull/728 -- same change, but with passing DCO check.

**Pull request closed without merging**